lib/checker.rb

```diff
@@ -6,8 +6,9 @@ def check(result, example)
     end
 
     def check_equal(result, records)
```

Comment:
The second parameter are not `records` anymore, since they also can include flags, memory positions, etc. We should change that name.